ykbh/src/lib.rs

```diff
@@ -0,0 +1,267 @@
+use std::alloc::{alloc, dealloc, Layout};
+use std::convert::{TryFrom, TryInto};
+use ykpack::{
+    self, Constant, ConstantInt, IPlace, Local, LocalDecl, Statement, Terminator, TypeId,
+    UnsignedInt,
+};
+use yktrace::sir::SIR;
+
+const SUSIZE: usize = std::mem::size_of::<usize>();
+
+pub struct SIRInterpreter {
+    map: *mut u8,
+    layout: Layout,
+    locals: Vec<usize>,
```

Comment:
```suggestion
    local_offsets: Vec<usize>,
```
It wasn't immediately clear to me that these are offsets into `map`.